tests/smoke/test_homepage.py

```diff
@@ -0,0 +1,33 @@
+"""Smoke tests for addons-frontend docker image."""
+
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    """Set up Firefox options."""
+    firefox_options.add_argument("-headless")
+    firefox_options.add_argument('-foreground')
+    return firefox_options
+
+
+@pytest.fixture
+def selenium(base_url, selenium):
+    """Set up selenium."""
+    # Simple wait for homepage to load
+    selenium.get(base_url)
+    WebDriverWait(selenium, 60).until(
+        EC.visibility_of_element_located(
+            (By.CSS_SELECTOR, ".visually-hidden")
+        )
+    )
+    return selenium
+
+
+@pytest.mark.nondestructive
+def test_hidden_class_is_found(selenium):
+    """Test hidden element is found."""
+    selenium.find_element(By.CSS_SELECTOR, ".visually-hidden")
```

Comment:
This is looking for `.visually-hidden` which is a class we use [all over the place](https://github.com/mozilla/addons-frontend/search?q=visually-hidden&unscoped_q=visually-hidden) so that won't work.

You mentioned that Selenium can't look for HTML comments but I found this: https://stackoverflow.com/questions/8416788/is-there-a-way-to-extract-html-comments-using-selenium-webdriver Maybe that could work?

Reply:
I would have to load it then parse it. Its not really a clean solution. What if I just look for the specific CSS path?

Reply:
Or I could add ```.do-not-remove``` to it, as it seems you only use that class for that comment

Reply:
How does finding an element by selector work? Surely that also loads and parses the content, no?